lms/static/js/student_account/views/AccessView.js

```diff
@@ -1,19 +1,11 @@
 var edx = edx || {};
 
-(function($, _, _s, Backbone, gettext) {
+(function($, _, _s, Backbone, History) {
     'use strict';
 
     edx.student = edx.student || {};
     edx.student.account = edx.student.account || {};
 
-    // Bind to StateChange Event
-    History.Adapter.bind( window, 'statechange', function() {
```

Comment:
As discussed via chat: JSHint was complaining about this code and it appears to not be doing anything. Let me know if you find out otherwise, and I can revert this since it's mostly an incidental change.